pages/githubissues.md

```diff
@@ -8,14 +8,18 @@ First, you will need to open an issue within the right repository ([upstream rep
 
 After this, you will then research the possible solutions or ideas on how to address the issue and write a comment with this information, as you can see [here](https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/15). Someone else will hopefully provide input (as Dogi did here) and then you can move ahead with making the proposed changes.
 
-Every time you start to work on an issue you need to create a branch to keep your work on various issues separate from each other. Before you create your branch, sync your repo following the process in the [previous step](http://open-learning-exchange.github.io/#!pages/gitandrepositories.md). Then use  `git checkout -b name_of_your_new_branch --track origin/name_of_your_new_branch` to create and switch to a new branch.
+Every time you start to work on an issue you need to create a branch to keep your work on various issues separate from each other. Before you create your branch, sync your repo following the process in the [previous step](http://open-learning-exchange.github.io/#!pages/gitandrepositories.md). Then use  `git checkout -b name_of_your_new_branch` to create and switch to a new branch on your local repository. Then create an initial commit `git commit -m "commit message"` and finally push the newly created local branch using `git push -u origin new-branch` to your remote repository.
+
+**TIP**: The default branch in your repository is called master. Keeping the master branch clean of any changes is a good practice so if you make any changes you dont want, you may revert back.
 
 Now you can go and make the proposed changes to your local files.
 
 **NOTE**: It's important that you are on the new branch, make sure you are on the correct branch with `git checkout branch-name` before committing. Use `git branch` to make sure you are on the correct branch.
 
 After you are done making your changes use the command `git status`. If the list of files changed only includes those you wish to modify, use the command `git add .` (The '.' is part of the command.) Otherwise you can also choose only certain changes to include by using `git add <file1> <file2> <file3>...`  Now that you have selected the files you wish to include, use the commands `git commit -m "commit message"` and `git push origin name_of_your_new_branch` to save your changes and push them to your Github. 
```

Comment:
Start line 19 with **Creating a new commit:**
Then, add the following to the last sentence:
and `git push origin name_of_your_new_branch` to save your changes and push them to your Github (or use `git push -u origin name_of_your_new_branch` if you use method 2 above and this is your first commit).

It is redundant, but at least we don't leave room for confusion! ;-)